src/com/missafternoon/budjen/Budjen.java

```diff
@@ -5,6 +5,13 @@
 
 public class Budjen {
     public static void main(String[] args) {
-        new Application(new DefaultTransactionData(), new DefaultLogger()).start();
+        String arg;
+        if (args.length == 0) {
+            arg = null;
+        }
+        else {
+            arg = args[0];
+        }
+        new Application(new DefaultTransactionData(), new DefaultLogger()).start(arg);
```

Comment:
I think perhaps it'll be easier going forward if `start` accepted `String[] args` as-is. E.g. adding transactions will probably require `start` handling multiple arguments.

This has the added benefit of minimising `main` as well. It does means that we'll have to do more logic in `start` however.